actions/databases.go

```diff
@@ -157,6 +159,19 @@ func DatabasesPost(c buffalo.Context) error {
 	}
 	if instanceOutput, err = rdsClient.Service.CreateDBInstanceWithContext(c, input.Instance); err != nil {
 		log.Println(err.Error())
+		if input.Cluster != nil {
+			// if this instance was in a new cluster, delete the cluster
+			log.Println("Deleting cluster", *input.Cluster.DBClusterIdentifier)
+			clusterInput := &rds.DeleteDBClusterInput{
+				DBClusterIdentifier: input.Cluster.DBClusterIdentifier,
+				SkipFinalSnapshot:   aws.Bool(true),
+			}
+			if _, errc := rdsClient.Service.DeleteDBClusterWithContext(c, clusterInput); errc != nil {
+				log.Println("Failed to delete cluster", errc.Error())
+			} else {
+				log.Println("Successfully requested deletion of cluster", *input.Cluster.DBClusterIdentifier)
+			}
+		}
```

Comment:
this is good - but i wonder if you you're going to do this if you need some exponential back off ... let's leave it now but pay attention.  you may be cleaning up too quickly and need to retry.